### DISCUSSION TITLE
resolution of the converted PNG file and flexibility of the html file for figure captions/documentation

### DISCUSSION THREAD

**Discussion by xiananjiang** [NOAA-GFDL/MDTF-diagnostics#232] [Ideas]
We are developing a multi-objective analysis POD at UCLA (David Neelin as the PI of this project, with Yi-Hung Kuo and others). For presentation of the POD generated figures, we are wondering if the following functions could be possibly considered to be built into the MDTF package?

1) To provide an option of different resolutions for the converted PNG files from the original PDF files by the POD? Currently, this conversion is automatically done by the MDTF package when the PDF files are stored into the model/PS directory, seems with low-resolutions. We want to have the high-resolution PNGs generated for better presentation in the HTML file.

2) To provide a template of the HTML file that we can put documentation/explanation of the PNG figures shown in the HTML file. Right now we have to put these notes along with the figure and converted to PNG files to be displayed on the web page. There are several limitations: a) the space on the figure file is usually very limited, which makes it difficult to have detailed explanation of the figures; b) after conversion to PNG, due to reduced resolution, these notes could become difficult to be read; c) the format for the notes with python plot is usually difficult to control, many times will be sensitive to different versions of python libraries; c) placing the documentation/explanation of the figures in the HTML file instead of in the figure files can improve the presentation of the products (e.g., with more selections for fonts, colors, etc).

Thanks!

**Comment by jkrasting** [comment]
Great suggestions @xiananjiang.  Thanks.   We are thinking of updating the html displays in the next phase, so we can certainly keep these in mind.
